v2/specification.md

```diff
@@ -110,16 +110,19 @@ Each payment option includes
 ## Payment Request
 
 ### Request
-A POST request should be made to the payment protocol url with `{chain, currency}`
+A POST request should be made to the payment protocol url with a JSON dictionary containing `{chain, currency}` fields
 
 #### Examples: 
 
- * /i/:someinvoiceid
+ * /i/someinvoiceid
 
 #### Headers
 * `Content-Type` = `application/payment-request`.
 * `x-paypro-version` = 2
 
+**Note: do NOT use the standard application/json `Content-Type`**
```

Comment:
When I view this file the formatting is a bit wonky, can you add an extra new line in be-tween these so they appear on separate lines.

Reply:
done